src/aus/api.test.js

```diff
@@ -1,4 +1,4 @@
-import { getUSPData } from './api';
+import { getUSPData } from './api.ts';
```

Comment:
Do we understand why these `.ts` have to be added? I think i did the same on my original PR for these changes but looking at other repos that use the eslint config it doesn't always seem necessary.

Reply:
I'm not sure. It seems to be because it's a js file importing a ts file. It also relates to this rule (https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/no-unresolved.md)

Reply:
Yes, this rule makes imports much more explicit 👍